pkgs/applications/audio/quodlibet/default.nix

```diff
@@ -1,136 +1,155 @@
-{ lib
-, fetchFromGitHub
-, tag ? ""
+{
+  lib,
+  fetchFromGitHub,
+  tag ? "",
 
   # build time
-, gettext
-, gobject-introspection
-, wrapGAppsHook3
+  gettext,
+  gobject-introspection,
+  wrapGAppsHook3,
 
   # runtime
-, adwaita-icon-theme
-, gdk-pixbuf
-, glib
-, glib-networking
-, gtk3
-, gtksourceview
-, kakasi
-, keybinder3
-, libappindicator-gtk3
-, libmodplug
-, librsvg
-, libsoup
-, webkitgtk
+  adwaita-icon-theme,
+  gdk-pixbuf,
+  glib,
+  glib-networking,
+  gtk3,
+  gtksourceview,
+  kakasi,
+  keybinder3,
+  libappindicator-gtk3,
+  libmodplug,
+  librsvg,
+  libsoup,
+  webkitgtk,
 
   # optional features
-, withDbusPython ? false
-, withMusicBrainzNgs ? false
-, withPahoMqtt ? false
-, withPyInotify ? false
-, withPypresence ? false
-, withSoco ? false
+  withDbusPython ? false,
+  withMusicBrainzNgs ? false,
+  withPahoMqtt ? false,
+  withPyInotify ? false,
+  withPypresence ? false,
+  withSoco ? false,
 
   # backends
-, withGstPlugins ? withGstreamerBackend
-, withGstreamerBackend ? true
-, gst_all_1
-, withXineBackend ? true
-, xine-lib
+  withGstPlugins ? withGstreamerBackend,
+  withGstreamerBackend ? true,
+  gst_all_1,
+  withXineBackend ? true,
+  xine-lib,
 
   # tests
-, dbus
-, glibcLocales
-, hicolor-icon-theme
-, python3
-, xvfb-run
+  dbus,
+  glibcLocales,
+  hicolor-icon-theme,
+  python3,
+  xvfb-run,
 }:
 
 python3.pkgs.buildPythonApplication rec {
   pname = "quodlibet${tag}";
   version = "4.6.0";
   format = "pyproject";
 
-  outputs = [ "out" "doc" ];
+  outputs = [
+    "out"
+    "doc"
+  ];
 
   src = fetchFromGitHub {
     owner = "quodlibet";
     repo = "quodlibet";
-    rev = "refs/tags/release-${version}";
-    hash = "sha256-dkO/CFN7Dk72xhtmcSDcwUciOPMeEjQS2mch+jSfiII=";
+    rev = "3dcf31dfc8db9806d1f73a47fdabc950d35ded1d";
+    hash = "sha256-8qWuxTvMF6ksDkbZ6wRLPCJK1cSqgGMPac/ht6qVpnA=";
```

Comment:
Just for reference, this PR switches to an unstable version of Quod Libet to make it work on Python 3.12, while https://github.com/NixOS/nixpkgs/pull/326924 patches the stable one (`4.6.0`).

Consider following [this](https://github.com/NixOS/nixpkgs/blob/eec16b48aa9e98c64b4fdd4a564092ea03a9c8a8/pkgs/README.md?plain=1#L352-L354) convention on what the value of the `version` attribute is in such cases.

Reply:
sure